projects/admin/src/app/record/detail-view/library-detail-view/exception-date/exception-date.component.html

```diff
@@ -14,11 +14,26 @@
    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<ng-container *ngIf="exception">
-  <i class="fa fa-circle {{ exception.is_open ? 'text-success': 'text-danger' }}" aria-hidden="true"></i>
-  {{ exception.start_date }}
-  <ng-container *ngIf="exception.end_date">
-  {{ 'date_to' | translate }} {{ exception.end_date }}
-  </ng-container>
-  : {{ exception.title }}
-</ng-container>
+<div *ngIf="exception" class="row my-2">
+  <i class="fa col-1" aria-hidden="true" [ngClass]="{
+    'fa-circle text-success': exception.is_open,
+    'fa-ban text-danger': !exception.is_open
+  }"></i>
+  <span class="col-3">
+    {{ exception.start_date }}
```

Comment:
Is this already formatted?

Reply:
according to https://github.com/rero/rero-ils-ui/blob/ecc65f4ef75b938bbec66832d676abf25e0f4cad/projects/admin/src/app/classes/library.ts#L50, it's already a formatted string.

It's formatted here https://github.com/rero/rero-ils-ui/blob/ecc65f4ef75b938bbec66832d676abf25e0f4cad/projects/admin/src/app/classes/library.ts#L243